forest/benchmarking/tests/test_direct_fidelity_estimation.py

```diff
@@ -1,28 +1,37 @@
-from forest.benchmarking.direct_fidelity_estimation import exhaustive_state_dfe, exhaustive_process_dfe, ratio_variance, \
-    monte_carlo_process_dfe
+from forest.benchmarking.direct_fidelity_estimation import exhaustive_state_dfe, exhaustive_process_dfe, \
+    monte_carlo_state_dfe, monte_carlo_process_dfe
+
 from pyquil import Program
 from pyquil.api import BenchmarkConnection
 from pyquil.gates import *
 from pyquil.numpy_simulator import NumpyWavefunctionSimulator
 from pyquil.operator_estimation import _one_q_state_prep
 
+from numpy.testing import assert_almost_equal
 
 def test_exhaustive_state_dfe(benchmarker: BenchmarkConnection):
     texpt = exhaustive_state_dfe(program=Program(X(0), X(1)), qubits=[0, 1],
                                  benchmarker=benchmarker)
-    assert len(texpt) == 2 ** 2 - 1
+    assert len(texpt.exp) == 2 ** 2 - 1
+
+
+def test_mc_state_dfe(benchmarker: BenchmarkConnection):
+    texpt = monte_carlo_state_dfe(program=Program(X(0), X(1), X(2), X(3), X(4), X(5), X(6)), 
+                         qubits=[0, 1, 2, 3, 4, 5, 6],
+                         n_terms=50, benchmarker=benchmarker)
+    assert len(texpt.exp) == 50
```

Comment:
To my earlier comment about abbreviating experiment in the dataclass definition--if we don't expand "experiment" then I think "expt" or "expts" might be a better abbreviation.